Gordon360/Models/ViewModels/ApartmentChoiceViewModel.cs

```diff
@@ -6,5 +6,17 @@ public class ApartmentChoiceViewModel
         public int ApplicationID { get; set; }
         public int HallRank { get; set; }
         public string HallName { get; set; }
+
+        public static implicit operator ApartmentChoiceViewModel(GET_AA_APARTMENT_CHOICES_BY_APP_ID_Result apartmentChoiceDBModel)
+        {
+            ApartmentChoiceViewModel vm = new ApartmentChoiceViewModel
+            {
+                ApplicationID = apartmentChoiceDBModel.AprtAppID,
+                HallRank = apartmentChoiceDBModel.Ranking,
+                HallName = apartmentChoiceDBModel.BLDG_CDE, //HallName = dbApartmentChoice.HallName, // Code for after we remade the AA_ApartmentChoices table
```

Comment:
Has this change been completed yet?

Reply:
Yes, it is completed and merged to apartapp-develop.

Reply:
Those comments existed in many other places, I left those for myself
I must have missed this one
```suggestion
                HallName = dbApartmentChoice.HallName, // Code for after we remade the AA_ApartmentChoices table
```

Reply:
This actually would have caused significant problems if it was merged before fixing this, so good catch 👍